tsconfig.json

```diff
@@ -1,4 +1,6 @@
 {
+  "$schema": "./tsconfig.schemastore-schema.json",
+  "ts-node": {},
```

Comment:
Is it empty for a reason?

Reply:
No reason, I forgot I added this.  I'll remove it.